src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -222,8 +222,10 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
   def createWorkflow(inputs: HostInputs): Future[Unit] = {
     val symbolStoreEntries = buildSymbolStoreEntries(workflow.namespace, inputs)
     symbolCache = symbolStoreEntries.groupBy(entry => SymbolCacheKey(entry.scope, entry.isInput))
+    // Currently assumes there is at most one possible final call, a `CopyWorkflowOutputs`.
+    val finalCall = workflow.workflowOutputsPath.toOption map { _ => CopyWorkflowOutputs(workflow) }
```

Comment:
I dislike the toOption here which silently swallows any error messages from the Try